data-sources/netlify-builds/config.yml

```diff
@@ -0,0 +1,29 @@
+id: netlify-builds
+
+displayName: Netlify Builds
+
+description: |
+  Netlify is an all-in-one platform for automating modern web projects. Replace your hosting infrastructure, continuous integration, and deployment pipeline with a single workflow. Integrate dynamic functionality like serverless functions, user authentication, and form handling as your projects grow.
+
+install:
+  primary:
+    nerdlet: nr1-install-newrelic.quickstart-installation-plan
+    nerdlet_state:
+      quickstart_id: 5861d7f5-26c0-43ad-bda8-c893c4b27b25
```

Comment:
I may have missed this while we were discussing the ticket, is this referencing back to the quickstart so that we can associate dashboards and alerts before we have the ability to directly reference connect them to the datasource?

Reply:
Correct! As a workaround for now, we are planning to have some of the data sources just send you through the quickstart install flow, hence the need for the associated quickstart ID.

Reply:
Yep!

This was something that Jerel and Jewels suggested as a short-term workaround. When a user goes to install a datasource, it will drop them into the install UI for the _quickstart_ so that it also installs the desired dashboards and alerts.

I tried to add some details to the ticket...but it was late in the day yesterday (well after the team had originally looked at and estimated it).

Reply:
Makes total sense! Thanks for the clarification.